packages/explat/src/assignment.ts

```diff
@@ -33,21 +42,47 @@ export const fetchExperimentAssignment = async ( {
 	 * });
 	 *
 	 */
-	const params = applyFilters( 'woocommerce_explat_request_args', {
-		experiment_name: experimentName,
-		anon_id: anonId ?? undefined,
-		woo_country_code:
-			window.wcSettings?.preloadSettings?.general
-				?.woocommerce_default_country ||
-			window.wcSettings?.admin?.preloadSettings?.general
-				?.woocommerce_default_country,
-	} );
-
-	const response = await window.fetch(
-		`https://public-api.wordpress.com/wpcom/v2/experiments/${ EXPLAT_VERSION }/assignments/woocommerce?${ stringify(
-			params
-		) }`
+	const queryString = stringify(
+		applyFilters( 'woocommerce_explat_request_args', {
+			experiment_name: experimentName,
+			anon_id: anonId ?? undefined,
+			woo_country_code:
+				window.wcSettings?.preloadSettings?.general
+					?.woocommerce_default_country ||
+				window.wcSettings?.admin?.preloadSettings?.general
+					?.woocommerce_default_country,
+		} )
 	);
 
-	return await response.json();
+	const response = await apiFetch( {
+		url: url && `${ url }?${ queryString }`,
+		path: path && `${ path }?${ queryString }`,
+	} );
+	return response;
 };
+
+export const fetchExperimentAssignment = async ( {
+	experimentName,
+	anonId,
+}: {
+	experimentName: string;
+	anonId: string | null;
+} ): Promise< unknown > =>
+	_fetchExperimentAssignment( {
+		url: `https://public-api.wordpress.com/wpcom/v2/experiments/${ EXPLAT_VERSION }/assignments/woocommerce?`,
```

Comment:
I think the `?` is extraneous

Reply:
<img width="1244" alt="image" src="https://user-images.githubusercontent.com/3747241/157684752-dea85782-d303-44e9-b02e-d00c05ec1f4a.png">

Reply:
Yes, you're right. Fixed in 3f8a0fdbc943fc848d5a8e362abea77d992b2b87.